example_github_workflow_files/automerge.yml

```diff
@@ -4,10 +4,18 @@ on:
   schedule:
     - cron: '05,17,29,41,53 * * * *'
   pull_request:
+    # labeled = run when labels are applied, so that the "Override AutoMerge: name similarity is okay label is respected.
```

Comment:
```suggestion
    # opened = run when the PR is first opened
    # labeled = run when labels are applied, so that the "Override AutoMerge: name similarity is okay label is respected.
```